src/itimer.cpp

```diff
@@ -6,11 +6,17 @@
 #include <sys/time.h>
 #include "itimer.h"
 #include "j9StackTraces.h"
+#include "log.h"
 #include "os.h"
 #include "vmEntry.h"
 
 
 Error ITimer::check(Arguments& args) {
+    Log::warn("DEPRECATED: The 'check' command is deprecated and will be removed in the next release.");
+    return Error::OK;
+}
+
+Error ITimer::start(Arguments& args) {
```

Comment:
`ITimer::start` did not call `check` previously.
In any case, these dummy `setitimer` calls are redundant, since `start()` already calls the real `setitimer`.